bagua-opentelemetry/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "bagua-opentelemetry"
+version = "0.1.0"
+edition = "2018"
+publish = ["private"]
+
+[dependencies]
+tracing = "0.1"
+async-std = { version = "1.6", features = ["attributes", "tokio1"] }
+async-trait = { version = "0.1" }
+hyper = { version = "0.14", features = ["full"] }
+opentelemetry = { version = "0.15", default-features = false, features = [
```

Comment:
why using a different runtime from `bagua-core-internal`'s opentelemetry?

Reply:
tokio does not work, removed